src/patterns/organisms/footer/footer-full.hbs

```diff
@@ -0,0 +1,51 @@
+---
+name: Footer with all options
+description: |
+  The optional elements are: logo, footer-sections (with links), language selector, and social media icons.
+notes: |
+  - It looks a little funny on the protocol page becuase the media queries expect it to be the full width of the page.
+links:
+    Footer Demo: /demos/footer.html
+---
+
+<footer class="mzp-c-footer">
+  <div class="mzp-l-content">
+    <nav class="mzp-c-footer-primary">
+      {{#embed "patterns.organisms.footer.includes.footer-primary-logo"}}{{/embed}}
+      <div class="mzp-c-footer-sections">
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+      </div>
+    </nav>
+
+    <nav class="mzp-c-footer-secondary">
+      {{#embed "patterns.organisms.footer.includes.footer-language"}}{{/embed}}
+      {{#embed "patterns.organisms.footer.includes.footer-links-social"}}{{/embed}}
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license">
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Content available under a <a rel="license" href="https://www.mozilla.org/foundation/licensing/website-content/">Creative Commons license</a>.
+        </p>
+        <ul class="mzp-c-footer-terms">
+          <li><a rel="nofollow" href="https://www.mozilla.org/privacy/websites/">Website Privacy Notice</a></li>
```

Comment:
We can remove `rel="nofollow"` on these links (@rraue removed them from the bedrock footer)